zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/KeystoneTokenProvider.java

```diff
@@ -16,6 +16,10 @@
   @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/oidc/auth")
   @Headers("Authorization: Bearer {token}")
   Response getKeystoneToken(@Param("token") String accessToken);
+  
+  @RequestLine("GET /v3/OS-FEDERATION/identity_providers/indigo-dc/protocols/iamoidc/auth")
```

Comment:
@RequestLine("GET /v3/OS-FEDERATION/identity_providers/{provider}/protocols/{protocol}/auth")
  @Headers("Authorization: Bearer {token}")
  Response getKeystoneToken(@Param("token")
                                String accessToken,
                            @Param("provider")
                                String provider,
                            @Param("protocol")
                                String protocol);`
`
I get the property from a file or from the CMDB if it's available.